Maquetado/login.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Book details</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+<body>
+  <form class="login-box">
+    <img class="login-box-logo" src="assets/LogoWolox.png" alt="Logo">
+    <div class="form-group">
+      <label class="login-box-text">Email</label>
+      <input class="login-box-input" type="email">
+    </div>
+    <div class="form-group">
+      <label class="login-box-text">Password</label>
+      <input class="login-box-input" type="password">
+    </div>
+    <div class="form-buttons">
+      <button class="login-box-btn-log">Login</button>
+      <hr class="login-box-divider">
```

Comment:
Maybe you can use a border-bottom here

Reply:
You can use `after` or `before` pseudo element